packages/backend/src/house/house.controller.ts

```diff
@@ -0,0 +1,41 @@
+import { Controller, Post, Body, Param, Get, Put, Query } from '@nestjs/common';
+import { UserStoreService } from '../db/user/userStore.service';
+import { HouseStoreService } from '../db/house/houseStore.service';
+import { CreateHouseDto } from './dto/create-house.dto';
+import { HouseUtil } from './house.util';
+
+@Controller('/api/v1/house')
+export class HouseController {
+  constructor(
+    private readonly houseStoreService: HouseStoreService,
+    private readonly userStoreService: UserStoreService,
+    private readonly houseUtil: HouseUtil,
+  ) {}
+
+  @Post()
+  async create(
+    @Body() createHouseDto: CreateHouseDto,
+    @Query('firebaseId') firebaseId: string,
```

Comment:
replace all instances of firebaseId/userId params with User decorator
```suggestion
    @User() user: DecodedIdToken,
```